App.tsx

```diff
@@ -278,7 +303,7 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                         <HeroBanner style={{ flex: 0, minWidth: 180, justifyContent: 'flex-end' }}>
                             <Hero
                                 label={'Temperature'}
-                                iconColor={PXBColors.black[500]}
+                                iconColor={theme.dark ? PXBColors.black[200] : theme.colors.text}
```

Comment:
same thought here...hero should be smart enough to choose the default color based on the theme type.

Reply:
Good catch here... I think I just added the logic because iconColor was previously being defined. Looks like the getColor function here works fine so I'll remove the iconColor prop definition from this instance.